numpy/random/_generator.pyx

```diff
@@ -717,6 +717,12 @@ cdef class Generator:
         out : bytes
             String of length `length`.
 
+        Notes
+        -----
+        This function generates random integers from a discrete uniform
+        distribution. It is independent of the from the CPU's native
```

Comment:
```suggestion
        distribution. The results are independent of the CPU's native
```
As written, "it" would refer to "this function", but I don't think that's what's intended.

We need to pick either "of the" or "from the", and I think "of the" is correct.

Reply:
But this still doesn't describe what the function does if the output is a string.